temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -222,6 +230,17 @@ public Scope getMetricsScope() {
    */
   public static class Builder {
 
+    // Default TLS protocol config that is used to communicate with temporal backend in the cloud.
```

Comment:
instead of "temporal backend in the cloud," can we say "TLS-enabled Temporal Server" instead? That seems more generic for OSS Client